homeassistant/components/homekit/__init__.py

```diff
@@ -68,7 +68,9 @@ def import_types():
     # pylint: disable=unused-variable
     from .covers import Window  # noqa F401
     # pylint: disable=unused-variable
-    from .sensors import TemperatureSensor # noqa F401
+    from .sensors import TemperatureSensor  # noqa F401
+    # pylint: disable=unused-variable
+    from .lights import Lightbulb  # noqa F401
```

Comment:
I think lightbulb is to specific since it would also represent a led strip among other things

Reply:
Will bear that in mind.

Reply:
"Lightbulb" is just the name of the "service" assigned by Apple. This shouldn't be the main concern right now.